lib/src/widgets/movie_backdrop_image.dart

```diff
@@ -10,36 +10,150 @@ class MovieBackdropImage extends StatelessWidget {
+            Text(
+              'NO LINK AVAILABLE',
+              key: ValueKey('textErrorImage'),
+              style: TextStyle(
+                color: Colors.grey.withOpacity(
+                  Dimension.genreCardOpacity,
+                ),
+                fontSize: Dimension.genreCardFontSize,
+              ),
+              textAlign: TextAlign.center,
+            ),
+          ],
+        );
+      }
+    }
+    return Container();
+  }
+
+  @override
+  Widget build(BuildContext context) {
     return Container(
       child: Stack(
         children: <Widget>[
           Container(
             width: double.infinity,
-            child: Hero(
-              tag: movieId,
-              child: backdropPath != null
-                  ? Image.network(
-                backdropPath,
-                fit: BoxFit.fill,
-                height: Dimension.movieInfoAppBarHeight,
-              )
-                  : Image.asset(
-                defaultImageRoute,
-                fit: BoxFit.contain,
-              ),
-            ),
+            child: Hero(tag: widget.movieId, child: _showChild()),
           ),
         ],
       ),
     );
   }
-
-  @override
-  Widget build(BuildContext context) {
-    return _buildContent();
-  }
 }
+
+/*
+showImage && _image != null
+                  ? Container(
+                      key: Key('backdropPathImage'),
+                      decoration: BoxDecoration(
+                        image: DecorationImage(image: _image.image, fit: BoxFit.fill),
+                      ),
+                      height: Dimension.movieInfoAppBarHeight,
+                    )
+                  : Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: <Widget>[
+                        showAsset
+                            ? Container(
+                                key: ValueKey('assetImage'),
+                                child: _imageAsset,
+                              )
+                            : IconButton(
+                                key: ValueKey('iconRouteEmpty'),
+                                icon: Icon(
+                                  Icons.error,
+                                  color: Colors.grey,
+                                ),
+                                iconSize: Dimension.iconSearchTypeSize,
+                              ),
+                        Text(
+                          'NO LINK AVAILABLE',
+                          key: ValueKey('textErrorImage'),
+                          style: TextStyle(
+                            color: Colors.grey.withOpacity(
+                              Dimension.genreCardOpacity,
+                            ),
+                            fontSize: Dimension.genreCardFontSize,
+                          ),
+                          textAlign: TextAlign.center,
+                        ),
+                      ],
+                    ),
+
+ */
```

Comment:
if you aren't using this code, remove it please

Reply:
Done